spec/models/solidus_easypost/customs_info_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe SolidusEasypost::CustomsInfo, :vcr do
```

Comment:
Consider adding comments that describe the rules being enforced.

Reply:
Will do